spec/factories/school_groups.rb

```diff
@@ -29,12 +29,14 @@
     uid { Faker::Number.number(digits: 5).to_s }
     url { Faker::Internet.url(host: "example.com") }
 
-    after(:build) do |school_group|
-      blank_image = File.open(Rails.root.join("spec/fixtures/files/blank_image.png"))
-      normalised_logo = ImageManipulator.new(image_file_path: blank_image.path).alter_dimensions_and_preserve_aspect_ratio("100", "100")
+    trait :with_logo_and_photo do
+      after(:build) do |school_group|
+        blank_image = File.open(Rails.root.join("spec/fixtures/files/blank_image.png"))
+        normalised_logo = ImageManipulator.new(image_file_path: blank_image.path).alter_dimensions_and_preserve_aspect_ratio("100", "100")
 
-      school_group.logo.attach(io: StringIO.open(normalised_logo.to_blob), filename: "logo.png", content_type: "image/png")
-      school_group.photo.attach(io: blank_image, filename: "photo.png", content_type: "image/png")
```

Comment:
Oh this was being set up for every School Group factory? :/ . Guess this change is bringing in some speed improvements!